### PR TITLE
Don't export or import the test contact

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/contacts/models.py [hunk 1]
```diff
@@ -16,10 +16,9 @@
 from smartmin.models import SmartModel
 from smartmin.csv_imports.models import ImportTask
 from temba.channels.models import Channel
-from temba.orgs.models import Org, OrgModelMixin, OrgEvent, OrgLock, ORG_DISPLAY_CACHE_TTL
```
Comment (rowanseymour) [inline temba/contacts/models.py]: Just some unused imports

File: temba/tests.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def clear_storage(self):
         """
         If a test has written files to storage, it should remove them by calling this
         """
-        shutil.rmtree('media/test_orgs')
+        shutil.rmtree('media/test_orgs', ignore_errors=True)
```
Comment (rowanseymour) [inline temba/tests.py]: So that it doesn't error if that folder doesn't exist

**Comment by norkans7** [comment]
I think errors confuses users, what about calling the the get_test_contact when we find test number in import file, so it will be considered as created?

**Comment by rowanseymour** [comment]
Maybe.. it's potentially confusing either way, though I suppose one could assume that users won't notice if they import 10 contacts from a spreadsheet and they only see 9 in the contacts list because one of them happened to use the test contact phone number. What if they've added additional field data? We can't update the test contact with that.

Ideally we'd give the users better feedback about errors during import, e.g.

Line | Problem
-----|------------
4    |  Invalid phone number
7    |  Phone number is reserved for testing

**Comment by norkans7** [comment]
yep, we keep this

**Comment by rowanseymour** [comment]
Well you raise a good point so we'll let @ericnewcomer weigh in before I merge this

**Comment by ericnewcomer** [comment]
It's OK to error on this case. We use an invalid number for test contacts. 

Only issue is that old exports that didn't error before now can. But I think that is acceptable in this case. Better to be explicit when something unusual is happening.

**Pull request merged**